Documentation/build-overview.rst

```diff
@@ -0,0 +1,16 @@
+Building SeisSol
```

Comment:
Can we change this to "Installing SeisSol"? Not something big but staying consistent with the line below would be nicer, IMO...

Reply:
+1